api/v1alpha1/jobset_webhook.go

```diff
@@ -14,18 +14,22 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"errors"
+	"fmt"
+
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/utils/pointer"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/jobset/pkg/util/util"
```

Comment:
This seems like a mistake?  double util package?

Reply:
So I took a look at Kueue and honestly seems that maybe you could name each function separately?  https://github.com/kubernetes-sigs/kueue/tree/main/pkg/util

And do you think you should test some of these?   You could leave it as an issue but not sure what coverage is targeted here.  My boss would get on me for having pure functions that aren't tested.  :(

Reply:
good idea, i will add unit tests for all these functions in a follow up PR.